src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.26.0
+
+11 November 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- **Hostname Length Limit**: Deploys now fail if a service hostname exceeds 63 characters, showing an error message. Previously, the dev environment deployed successfully even if the endpoint didn’t work. This may affect environments that deployed without issues before.
+- **Helm Release Name Limit**: Helm release names are now limited in length. Use defaultBackend.nameOverride to shorten the default backend service name if needed. If changing this during an upgrade, [follow the guide as it may impact the installation](self-hosted/helm-configuration.mdx#manual-migration-steps-when-renaming-the-defaultbackend-service).
+- **Private Repository Deploys**: Deploying private repositories now uses the Okteto backend as the SSH agent, rather than mounting the local SSH agent. This change ensures feature parity between remote and local deploys but may impact scenarios where private repositories are cloned as part of commands defined in the deploy section during remote execution.
+- **Buildkit Persistence Enabled**: Buildkit persistence is now enabled by default, with a 100Gi disk and cache set to 90% of the disk size. If you previously used buildkit.persistence.cache, adjust to the new ratio, as this setting is no longer applicable.
+
+### New Features
+
+- **[Introducing the Okteto Resource Manager](admin/resource-manager.mdx)**: a new feature that automatically optimizes CPU and memory requests for your environments. By analyzing real-time resource utilization, the Resource Manager dynamically adjusts resource requests to ensure efficient usage, prevent node overload, and improve overall cluster performance. This feature simplifies resource management, reduces manual adjustments, and enhances application stability, especially in larger clusters. The default installation provides recommendations but doesn't apply them automatically. [See our docs for details on how to apply these automatically](admin/resource-manager.mdx).
+- Added Okteto [Garbage Collector settings to the Admin Dashboard](admin/cleanup.mdx): Admins can now manage sleep and delete periods from the Dashboard, with the ability to set different configurations for Namespaces and Preview Environments
+- [Remote Execution can now be made the default](core/remote-execution.mdx) by configuring it in the Okteto Admin Dashboard. This improves usability, particularly for private Git repositories and context synchronization with BuildKit, making remote deployments more consistent and reliable
```

Comment:
I would rethink this paragraph. There are two things here:
- Having this option gives admin the power to force remote deploys everywhere. Remote deploys is our recommendation to have a more consistent deploy behavior across developers
- We have implemented two improvements on deploy on remote to provide feature parity with local deploys. They are `context` and the ability to close private git repositories as part of the deploy commands

Reply:
Does this sound ok?
[Remote Execution can now be set as the default](core/remote-execution.mdx) in the Okteto Admin Dashboard, allowing admins to enforce consistent remote deploys. Improvements include feature parity with local deploys, such as context synchronization and private Git repository cloning during deploys

Reply:
Looks good!